.github/workflows/csslint.yml

```diff
@@ -0,0 +1,3 @@
+name: Stylesheet Lint
```

Comment:
@andyderek see #73 esp https://github.com/Xcov19/mycovidconnect/pull/73/checks?check_run_id=1172342856 for why its failing.

See https://github.com/Xcov19/mycovidconnect/blob/1427edb544c2505e90186f7212909382e8023611/.github/workflows/csslint.yml for how ive added actions

Reply:
@andyderek if the yml file is not added like this, it won't run in the pipeline. See the pipeline checks below

Reply:
I was just trying to see why it cannot find the css files.  That’s what it’s saying in the errors.

Reply:
you need to check how ive added in the workflow to make it work in this PR; 
the css patterns arent matching